### PR TITLE
Add db name to the column name for `show tables` statement

### PR DESCRIPTION
**Pull request by jennifersp**
- Database name is now included in column name in SHOW TABLES statement
- Fixed `cardinality` column of `information_schema.statistics` table returned wrong type before

### CODE REVIEW

File: sql/parse/parse_test.go [hunk 1]
```diff
@@ -1680,67 +1680,60 @@ CREATE TABLE t2
...
 	`SHOW FULL TABLES FROM bar AS OF 'abc' LIKE 'foo'`: plan.NewFilter(
 		expression.NewLike(
-			expression.NewUnresolvedColumn("Table"),
+			expression.NewUnresolvedColumn("Tables_in_bar"),
 			expression.NewLiteral("foo", sql.LongText),
 			nil,
 		),
 		plan.NewShowTables(sql.UnresolvedDatabase("bar"), true, expression.NewLiteral("abc", sql.LongText)),
 	),
-	"SHOW FULL TABLES FROM bar WHERE `Table` = 'foo'": plan.NewFilter(
```
Comment (zachmu) [inline sql/parse/parse_test.go:1737]: This is a parse test and is still valid (it just won't run correctly).

Comment (zachmu) [reply sql/parse/parse_test.go:1737]: Just change the column name, don't delete the test

File: sql/information_schema/information_schema.go [hunk 2]
```diff
@@ -1033,9 +1032,15 @@ func statisticsRowIter(ctx *Context, c Catalog) (RowIter, error) {
...
-								if err != nil {
-									return nil, err
+								c, cErr := st.NumRows(ctx)
+								if cErr != nil {
+									return nil, cErr
+								}
+								// cardinality is int64 type, but NumRows return uint64
+								// so casting it to int64 with a check for negative number
+								cardinality = int64(c)
+								if cardinality < 0 {
```
Comment (VinaiRachakonda) [inline sql/information_schema/information_schema.go:1042]: Abstract this out to a separate method. Code is getting a little messy

File: sql/plan/show_tables.go [hunk 3]
```diff
@@ -76,11 +68,20 @@ func (*ShowTables) Children() []sql.Node {
 
 // Schema implements the Node interface.
 func (p *ShowTables) Schema() sql.Schema {
+	var sch sql.Schema
+	colName := fmt.Sprintf("Tables_in_%s", p.Database().Name())
 	if p.Full {
-		return showTablesFullSchema
+		sch = sql.Schema{
```
Comment (VinaiRachakonda) [inline sql/plan/show_tables.go:74]:
```go
sch = sql.Schema{{Name: colName, Type: sql.LongText}}
if p.full {
  sch = append(sch, {Name: "Table_type", Type: sql.LongText})
}

```

**Review by VinaiRachakonda — APPROVED** [review]
LGTM

**Pull request merged**